### DISCUSSION TITLE
docker compose部署完后无法访问

### DISCUSSION THREAD

**Discussion by CCCAUCHY** [DIYgod/RSSHub#12088] [Q&A]
无法访问我的服务器，按照[教程](https://docs.rsshub.app/install/#docker-compose-bu-shu)部署的。
不管从公网访还是内部都无法访问
```
telnet 127.0.0.1 80
Trying 127.0.0.1...
Connected to 127.0.0.1.
Escape character is '^]'.
Connection closed by foreign host.
```

编辑：
port改成80:1200后就可以了，本项目不能监听非1200的端口吗？

**Comment by TonyRL** [comment]
Please take a good at look https://docs.rsshub.app/install/
You can change the default listening port to any other value
